README.md

```diff
@@ -40,6 +40,8 @@ privacy_engine.attach(optimizer)
 ```
 The [MNIST example](examples/mnist.py) contains an end to end run.
 
+The [compute_dp-sgd_privacy](torchdp/compute_dp-sgd_privacy.py) command-line script computes privacy of a model trained with DP-SGD.
+
```

Comment:
Because file name has been changed...
```suggestion
The [compute_dp_sgd_privacy](torchdp/compute_dp_sgd_privacy.py) command-line script computes privacy of a model trained with DP-SGD.

```